modules/nodes/variables.tf

```diff
@@ -101,7 +101,8 @@ variable "eks_info" {
     }
   EOF
   type = object({
-    k8s_pre_setup_sh_file = string
+    k8s_pre_setup_change_hash = string
```

Comment:
These need to be optional, otherwise the plans will fail, due to the outputs not being updated for infra|eks at plan time.